### ISSUE TITLE
ucaresystem-core resets terminal scrollback

### ISSUE THREAD

**Issue by f2404** [Utappia/uCareSystem#19]
When running `ucaresystem-core` in a terminal that has some scrollback, I'm losing it. Would be nice to keep it.

**Comment by cerebrux** [comment]
@f2404 Thanks for the report.
What terminal do you use ?
When you run ucaresystem-core in the terminal, it automatically scrolls down till it finishes all the jobs. Once it is finished you can scroll buck up to see the outputs.

**Comment by f2404** [comment]
I've tried xfce4-terminal (vte based) and xterm.

The following is showing the xterm experience. My terminal size is the default - 80x25.

To show the problem, I first run a command to fill in scrollback; after running it, my terminal looks as follows (all rows are filled):
```
igor@ubuntu:~$ for i in `seq 1 22`; do echo $i; done
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
igor@ubuntu:~$ 
```

Then I run `sudo ucaresystem-core`. After I scroll all the way up, everything I can see is
```
_______________________________________________________
                                                       
            uCareSystem Core 4.4.0            
            ~~~~~~~~~~~~~~~~~~~~~                      
                                                       
 Welcome to all-in-one System Update and maintenance   
 assistant app.
... [ucaresystem output continues] ...
```

As you can see, all my scrollback is lost after I've run ucaresystem-core.

**Comment by cerebrux** [comment]
Interesting...
This needs investigation. 

Can you please click the ucaresystem-core icon from the application menu and tell me if the same happens with the default ucaresystem-core terminal that automatically launches ?

**Comment by f2404** [comment]
Ucaresystem-core menu item launches xterm on my system (though xfce4-terminal should be the default terminal emulator, I believe).
Anyway, I don't think this experiment is relevant to the problem as there's no scrollback history that can be lost - since a clean new terminal is started when I run ucaresystem-core from the menu. What I see in it is
```
_______________________________________________________
                                                       
            uCareSystem Core 4.4.0            
            ~~~~~~~~~~~~~~~~~~~~~
```
and so on.

**Comment by f2404** [comment]
If I comment out this [clear](https://github.com/Utappia/uCareSystem/blob/master/ucaresystem-core#L38) call, my scrollback gets preserved.
Please consider removing this call.

**Comment by cerebrux** [comment]
Awesome ! 
I will test it on various terminals (as I need to support a ton of Desktop Environments)
and I will push an update .

Thank you so much taking the time to report and find a workaround.

**Comment by f2404** [comment]
Thank you!

**Comment by cerebrux** [comment]
Fixed with 88340cb (develop). soon will be pushed to master (stable)
